types/chrome/index.d.ts

```diff
@@ -1556,7 +1562,7 @@ declare namespace chrome.contextMenus {
         /** Optional. The text to be displayed in the item; this is required unless type is 'separator'. When the context is 'selection', you can use %s within the string to show the selected text. For example, if this parameter's value is "Translate '%s' to Pig Latin" and the user selects the word "cool", the context menu item for the selection is "Translate 'cool' to Pig Latin".  */
         title?: string | undefined;
         /** Optional. List of contexts this menu item will appear in. Defaults to ['page'] if not specified.  */
-        contexts?: string[] | undefined;
+        contexts?: ContextType[] | undefined;
```

Comment:
The docs specify that we should be able to use a ContextType here, along with a array of ContextType's.
```suggestion
        contexts?: ContextType | ContextType[] | undefined;
```